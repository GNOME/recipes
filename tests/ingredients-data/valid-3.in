2.123	l	wine	
1/2	g	nutmeg	
