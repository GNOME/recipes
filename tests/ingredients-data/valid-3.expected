AMOUNT 2.123000
UNIT l
NAME Wine

AMOUNT 0.500000
UNIT g
NAME Nutmeg

